library/aws/shared.ts

```diff
@@ -45,6 +45,7 @@ export interface AvailabilityZone {
  */
 export interface AzResourcesExternalSubnet {
   external_subnet: AR.Subnet;
+  external_route_table_id: AR.RouteTableId;
```

Comment:
It could be better to say `external_route_table: AR.RouteTable;`
but a bit more work - so dont worry about it.